manifest.yaml

```diff
@@ -1,7 +1,7 @@
 id: jellyfin
 title: "Jellyfin"
-version: 10.8.9.1
-release-notes: "Minor fixes"
+version: 10.8.9.2
+release-notes: "Minor fixes and refactoring"
```

Comment:
```suggestion
release-notes: "Initial release of Jellyfin for StartOS"
```

Reply:
I'll leave this to your discretion, as I'm not sure what the recommended best practice is here

Reply:
Per our discussion, this has been updated to `initial release` in 1be02f14e9672158205a3fcb04490a4bbae3d993